XApp.d.ts

```diff
@@ -0,0 +1,8 @@
+declare namespace imports.gi.XApp {
```

Comment:
Can you remove this file? I added the generation from .gir file for it for you

Reply:
Done. That's obviously much better :-)